metasource/driver/move.go

```diff
@@ -0,0 +1,33 @@
+package driver
+
+import (
+	"fmt"
+	"log/slog"
+	"os"
+	"strings"
+)
+
+func MoveGeneratedFiles(loca string) error {
```

Comment:
Also, for the sake of simplicity

```suggestion
func TransferResult(loca string) error {
```